src/modules/launcher/Plugins/Wox.Plugin.Program/Programs/Win32.cs

```diff
@@ -462,7 +470,15 @@ public static Win32[] All(Settings settings)
                     programs = programs.Concat(startMenu);
                 }
 
-                return programs.ToArray();
+                var programsWithoutLnk = programs.Where(x => string.IsNullOrEmpty(x.LnkResolvedPath));
```

Comment:
Just for clarity could you comment about what causes LnkResolvedPath to be null or empty?

Reply:
i added this in to another thread.  Only time Lnk path is filled is when it was pulled from LNK